### PR TITLE
Cosmetic adjustments in syntax

### PR DESCRIPTION
**Pull request by kpbochenek**
mainly indenting match-case code:
```scala
expr match {
  case cond =>
     expr1   // indenting it to not align with case
}
```
Looks more like dotty where case block must be indented.

### CODE REVIEW

File: compiler/src/dotty/tools/backend/jvm/BCodeBodyBuilder.scala [hunk 1]
```diff
@@ -862,8 +858,7 @@ trait BCodeBodyBuilder extends BCodeSkelBuilder {
      *
      * On a second pass, we emit the switch blocks, one for each different target.
      */
-    private def genMatch(tree: Match): BType = tree match {
-      case Match(selector, cases) =>
+    private def genMatch(tree: Match): BType = tree match { case Match(selector, cases) =>
```
Comment (tgodzik) [inline compiler/src/dotty/tools/backend/jvm/BCodeBodyBuilder.scala:861]: Can we revert this change?

File: compiler/src/dotty/tools/backend/jvm/BCodeSyncAndTry.scala [hunk 2]
```diff
@@ -176,8 +176,7 @@ trait BCodeSyncAndTry extends BCodeBodyBuilder {
      *    - "exception-handler-version-of-finally-block" respectively.
      *
      */
-    def genLoadTry(tree: Try): BType = tree match {
-      case Try(block, catches, finalizer) =>
+    def genLoadTry(tree: Try): BType = tree match { case Try(block, catches, finalizer) =>
```
Comment (tgodzik) [inline compiler/src/dotty/tools/backend/jvm/BCodeSyncAndTry.scala:179]: Also this?

File: compiler/src/dotty/tools/backend/jvm/BCodeBodyBuilder.scala [hunk 3]
```diff
@@ -599,8 +596,7 @@ trait BCodeBodyBuilder extends BCodeSkelBuilder {
       }
     } // end of genReturn()
 
-    def genWhileDo(tree: WhileDo): BType = tree match{
-      case WhileDo(cond, body) =>
+    def genWhileDo(tree: WhileDo): BType = tree match { case WhileDo(cond, body) =>
```
Comment (tgodzik) [inline compiler/src/dotty/tools/backend/jvm/BCodeBodyBuilder.scala:599]: Can we revert this?

**Review by tgodzik — CHANGES_REQUESTED** [review]
The changes here are only cosmetic from what I see, but I would revert a couple of weird formatting choices.

**Comment by nicolasstucki** [comment]
We should not do these formatting changes on the backend. It will make it harder to port commits from the scala2 backend as it will make the diff much harder to read. We should only do this once the backend is up to date.

**Review by nicolasstucki — CHANGES_REQUESTED** [review]
Please revert the changes to the backend. Sorry about that.

**Comment by kpbochenek** [comment]
okay I will close this and move other changes to different PR.

**Pull request closed without merging**